test/support/oauthUtil.js

```diff
@@ -242,7 +259,11 @@ oauthUtil.setup = function(opts) {
         return;
       }
       var expectedResp = opts.expectedResp || defaultResponse;
-      validateResponse(res, expectedResp);
+      if (opts.validateFunc) {
+        opts.validateFunc(res);
```

Comment:
Is `validateFunc` a new/separate feature from the other "lifted" features? Does it need to be added to the README, CHANGELOG?

Reply:
it's a test only function.